sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeBatchActionsOperation.cs

```diff
@@ -262,7 +262,8 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     _actionSucceeded = update.Value.Tasks.Completed;
                     _totalActions = update.Value.Tasks.Total;
 
-                    if (update.Value.Status == TextAnalyticsOperationStatus.Succeeded || update.Value.Status == TextAnalyticsOperationStatus.PartiallyCompleted)
+                    // TODO - Remove PartiallySucceeded once service deploys this to WestUS2
```

Comment:
Added this TODO, to remove `update.Value.Status == TextAnalyticsOperationStatus.PartiallySucceeded` on Friday.

Reply:
create an issue also to track it in case we forget